lib/logging.dart

```diff
@@ -64,7 +64,7 @@ class Logger {
   /// It can be useful when you just need a local short-living logger,
   /// which you'd like to be garbage-collected later.
   factory Logger.detached(String name) {
-    return new Logger._internal(name, null, new Map<String, Logger>());
+    return new Logger._internal(name, null!, new Map<String, Logger>());
```

Comment:
scary...

Reply:
Yeah.  See my other comment on this line.